tests/e2e/SMS/Msg91Test.php

```diff
@@ -12,19 +12,17 @@ class Msg91Test extends Base
      */
     public function testSendSMS()
     {
-        // $sender = new Msg91(getenv('MSG_91_SENDER_ID'), getenv('MSG_91_AUTH_KEY'));
+        $sender = new Msg91(getenv('MSG_91_SENDER_ID'), getenv('MSG_91_AUTH_KEY'));
 
-        // $message = new SMS(
-        //     to: [getenv('MSG_91_TO')],
-        //     content: 'Test Content',
-        //     from: getenv('MSG_91_FROM')
-        // );
+        $message = new SMS(
+            to: [getenv('MSG_91_TO')],
+            content: 'Test Content',
+            from: getenv('MSG_91_FROM')
+        );
 
-        // $response = $sender->send($message);
-        // $result = \json_decode($response, true);
+        $response = $sender->send($message);
+        $result = \json_decode($response, true);
 
-        // $this->assertEquals('success', $result['type']);
-
-        $this->markTestSkipped('Msg91 requires business verification to use template and SMS api.');
+        $this->assertEquals('success', $result['type']);
```

Comment:
This test is fixed now we are verified

Reply:
https://api.msg91.com/apidoc/textsms/send-sms-flow.php
I removed this check since the `message` property is not really the content instead some sort of an ID that we cannot use for assertion